addon/components/ember-wrapper.js

```diff
@@ -0,0 +1,31 @@
+import React from 'npm:react';
+
+/**
+ * A React component that is used to render HTML Nodes.
+ *
+ * The primary usage of this component is to support `props.children` in `react-component`.
+ * The catch is that an additional `<span>` is introduced as a wrapper.
+ *
+ * Note that although this is possible, it should be used as a tool to migrate Ember to React
+ * without the hard requirement to start with leaf components. It is highly recommended to
+ * have clean React component tree whenever possible.
+ *
+ * Integration guide: https://reactjs.org/docs/integrating-with-other-libraries.html
+ */
+class EmberWrapper extends React.Component {
```

Comment:
I think we could name this more clearly. Maybe `react-component-yield-wrapper` or even `react-component/yield-wrapper`

Reply:
Sounds good. I was confused by myself too that `EmberWrapper` is actually a React component. `yield-wrapper` sounds more intuitive